experimental/devices/ads1x15/ads1x15.go

```diff
@@ -0,0 +1,392 @@
+package ads1x15
+
+import (
+	"encoding/binary"
+	"errors"
+	"math"
+	"sync"
+	"time"
+
+	"periph.io/x/periph/conn/physic"
+	"periph.io/x/periph/conn/pin"
+
+	"fmt"
```

Comment:
Please use `goimports -w` instead of managing imports yourself, this means grouping the stdlib imports first (including fmt), then all periph.io imports in a second group.

Reply:
Oups, sorry